pandas/core/groupby/grouper.py

```diff
@@ -59,7 +59,7 @@ class Grouper:
         <http://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#offset-aliases>`_.
     axis : number/name of the axis, defaults to 0
     sort : bool, default to False
-        whether to sort the resulting labels
+        whether to sort the resulting codes
```

Comment:
Should this remain as labels? I think our sorting is actually done that way and not off of any internal codes

Reply:
Yeah, you're right, I'll revert that. As a sidenote, it sort by calling ``self.values`` so it's very slow. I think it should be changed to be based on codes (future PR, but that breaks the API, which isn't good after 1.0?).